### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by reneeteng**
Translation done. Links need to be updated later.

### CODE REVIEW

File: content/zh/docs/upgrade/_index.md [hunk 1]
```diff
@@ -11,24 +11,24 @@ icon: "/images/docs/docs.svg"
 
 ---
 
-This chapter demonstrates how cluster operators can upgrade existing KubeSphere to v3.0.0.
+本章将演示集群运营商如何将现有的 KubeSphere 升级到 v3.0.0。
```
Comment (Sherlock113) [inline content/zh/docs/upgrade/_index.md:14]: cluster operators here mean the person responsible for the operation and management of clusters.
->
本章演示集群管理员如何将现有的 KubeSphere 升级到 v3.0.0。

File: content/zh/docs/upgrade/_index.md [hunk 2]
```diff
@@ -11,24 +11,24 @@ icon: "/images/docs/docs.svg"
...
 
-## [Overview](../upgrade/upgrade-overview/)
+## [总览](../upgrade/upgrade-overview/)
```
Comment (Sherlock113) [inline content/zh/docs/upgrade/_index.md:16]: 总览 -> 概述

File: content/zh/docs/upgrade/_index.md [hunk 3]
```diff
@@ -11,24 +11,24 @@ icon: "/images/docs/docs.svg"
...
+## [使用 KubeKey 升级](../upgrade/upgrade-with-kubekey/)
 
-Follow steps to use KubeKey to upgrade Kubernetes and KubeSphere.
+按照步骤使用 KubeKey 升级 Kubernetes 和 KubeSphere。
 
-## [Upgrade with ks-installer](../upgrade/upgrade-with-ks-installer/)
+## [使用 ks-installer 升级](../upgrade/upgrade-with-ks-installer/)
 
-Follow steps to use ks-installer to upgrade KubeSphere.
+按照步骤使用 ks-installer 升级KubeSphere。
```
Comment (Sherlock113) [inline content/zh/docs/upgrade/_index.md:26]: 升级KubeSphere -> 升级 KubeSphere

**Review by Sherlock113 — COMMENTED** [review]
Please have a look at my comments. Thanks.

File: content/zh/docs/upgrade/_index.md (hunk 3 above)
Comment (reneeteng) [reply content/zh/docs/upgrade/_index.md:26]: OK.

File: content/zh/docs/upgrade/_index.md (hunk 2 above)
Comment (reneeteng) [reply content/zh/docs/upgrade/_index.md:16]: OK

File: content/zh/docs/upgrade/_index.md (hunk 1 above)
Comment (reneeteng) [reply content/zh/docs/upgrade/_index.md:14]: OK

**Pull request merged**